string teamthrow
int32  aruco_name_throw
---
